selfdrive/car/hyundai/carcontroller.py

```diff
@@ -62,24 +65,48 @@ def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, visual_alert,
                         left_lane, right_lane, left_lane_depart, right_lane_depart)
 
     can_sends = []
+
+    # tester present - w/ no response (keeps radar disabled)
+    if CS.CP.openpilotLongitudinalControl:
+      if (frame % 10) == 0:
+        can_sends.append([0x7D0, 0, b"\x02\x3E\x80\x00\x00\x00\x00\x00", 0])
+
     can_sends.append(create_lkas11(self.packer, frame, self.car_fingerprint, apply_steer, lkas_active,
                                    CS.lkas11, sys_warning, sys_state, enabled,
                                    left_lane, right_lane,
                                    left_lane_warning, right_lane_warning))
 
-    if pcm_cancel_cmd:
-      can_sends.append(create_clu11(self.packer, frame, CS.clu11, Buttons.CANCEL))
-    elif CS.out.cruiseState.standstill:
-      # send resume at a max freq of 10Hz
-      if (frame - self.last_resume_frame) * DT_CTRL > 0.1:
-        # send 25 messages at a time to increases the likelihood of resume being accepted
-        can_sends.extend([create_clu11(self.packer, frame, CS.clu11, Buttons.RES_ACCEL)] * 25)
-        self.last_resume_frame = frame
+    if not CS.CP.openpilotLongitudinalControl:
+      if pcm_cancel_cmd:
+        can_sends.append(create_clu11(self.packer, frame, CS.clu11, Buttons.CANCEL))
+      elif CS.out.cruiseState.standstill:
+        # send resume at a max freq of 10Hz
+        if (frame - self.last_resume_frame) * DT_CTRL > 0.1:
+          # send 25 messages at a time to increases the likelihood of resume being accepted
+          can_sends.extend([create_clu11(self.packer, frame, CS.clu11, Buttons.RES_ACCEL)] * 25)
+          self.last_resume_frame = frame
+
+    if frame % 2 == 0 and CS.CP.openpilotLongitudinalControl:
+      lead_visible = True
+      accel = actuators.accel if enabled else 0
+      accel = clip(accel, CarControllerParams.ACCEL_MIN, CarControllerParams.ACCEL_MAX)
+
+      stopping = actuators.longControlState == LongCtrlState.stopping
```

Comment:
I added requiring `CS.out.vEgo <= 0.1` when setting this for my vehicle because otherwise I would sometimes get a really hard, uncomfortable jerk coming to a stop